lib/manager/components/HomeProjectDropdown.js

```diff
@@ -2,9 +2,9 @@
 
 import Icon from '@conveyal/woonerf/components/icon'
 import React, {Component} from 'react'
-import { Button, DropdownButton, MenuItem } from 'react-bootstrap'
+import { Button, MenuItem } from 'react-bootstrap'
 import { LinkContainer } from 'react-router-bootstrap'
-import {getAbbreviatedProjectName} from '../../common/util/util'
```

Comment:
Without this method things end up like this: 
![image](https://user-images.githubusercontent.com/2370911/115867386-0c1fbe80-a409-11eb-8aef-bdd225ad534c.png)